views/templates/front/payment.tpl

```diff
@@ -43,5 +43,9 @@
         <input type="hidden" name="onpay_language" value="{$form_fields['onpay_language']}">
     {/if}
 
+    {foreach from=$form_info_fields key=field item=value}
+        <input type="hidden" name="{$field}" value="{$value}">
+    {/foreach}
+
```

Comment:
Again, this should not be necessary. And will actually just result in double rendering of the fields.